helpers/loanHelpers/userCollection.tsx

```diff
@@ -213,6 +114,22 @@ export async function calcNFT(
     return 0;
   }
 }
+
+/**
+ * @description pollutes the chart on lend with dummy historic rates
+ * @params none
+ * @returns chart data
+ */
+const getPositionData = () => {
```

Comment:
is this relevant? dummy chart data?

Reply:
Is required for future work - we should leave it there for now.